### PR TITLE
Fix typo in docs

### PR DESCRIPTION
**Pull request by VMRuiz**
It ain't much, but it's honest work

### CODE REVIEW

File: docs/providers.rst [hunk 1]
```diff
@@ -59,7 +59,7 @@ Please, check the docs provided in the following API reference for more details:
 :class:`~.PageObjectInputProvider`.
 
 
-Cache Suppport in Providers
+Cache Support in Providers
 ===========================
```
Comment (wRAR) [inline docs/providers.rst:63]:
```suggestion
==========================
```

Comment (wRAR) [reply docs/providers.rst:63]: :)

Comment (VMRuiz) [reply docs/providers.rst:63]: One line change with a new one line bug! Of course.

**Review by wRAR — APPROVED** [review]

**Review by Gallaecio — APPROVED** [review]
Appproved!

**Pull request merged**